test/cli/no-stdlib/no-stdlib.out

```diff
@@ -3,4 +3,8 @@ No errors! Great job.
 -e:1: Method `puts` does not exist on `T.class_of(<root>)` https://srb.help/7003
      1 |puts 'Hello > World'
         ^^^^^^^^^^^^^^^^^^^^
+  Got T.class_of(<root>) originating from:
+    -e:1: Possibly uninitialized (`NilClass`) in:
```

Comment:
Our error printing make this show up as a weird loc, because it just so happens that the loc we use for `<self>` at the file top level is also the same loc we use for pointing to uninitialized variables.